src/selectors/configmap/sc-defaults.js

```diff
@@ -0,0 +1,13 @@
+import { get } from 'lodash';
+
+import { PVC_ACCESSMODE_DEFAULT, PVC_VOLUMEMODE_DEFAULT } from '../../constants';
+
+export const getDefaultSCAccessMode = (storageClassConfigMap, scName) => {
+  const defaultAccessMode = get(storageClassConfigMap, ['data', 'accessMode'], PVC_ACCESSMODE_DEFAULT);
+  return get(storageClassConfigMap, ['data', `${scName}.accessMode`], defaultAccessMode);
```

Comment:
is the property really named `${scName}.accessMode` with a dot inside?

Reply:
yes